packages/roosterjs-editor-core/lib/coreApi/selectImage.ts

```diff
@@ -21,3 +30,38 @@ export const selectImage: SelectImage = (image: HTMLImageElement | null) => {
 
     return null;
 };
+
+const select = (core: EditorCore, image: HTMLImageElement) => {
```

Comment:
Is there any duplicated code with table selection? we can move them out into dom package under style folder

Reply:
I added this duplicated code as function under utils folder of the core, it add styles, but only for selection and it uses editor core object.